integration-cli/docker_cli_daemon_test.go

```diff
@@ -481,6 +481,31 @@ func TestDaemonUpgradeWithVolumes(t *testing.T) {
 	logDone("daemon - volumes from old(pre 1.3) daemon work")
 }
 
+// GH#11320 - verify that the daemon exits on failure properly
+func TestDaemonExitOnFailure(t *testing.T) {
```

Comment:
sure, no problem.  I only picked -b and --bip because I need something that will cause error to be returned on daemon init.  There are others, but this seemed like a reasonable choice.